develop/styleguide/python.rst

```diff
@@ -66,11 +65,16 @@ Based on code we love to look at (Pyramid, Requests, etc.), we allow the followi
            "an_exceptionally_long_string_of_characters",
        ]
 
- * Arguments on first line, directly after the opening parenthesis are forbidden when breaking lines.
- * The last argument line needs to have a trailing comma (to be nice to the next developer coming in to add something as an argument and minimize VCS diffs in these cases).
- * The closing parenthesis or bracket needs to have the same indentation level as the first line.
- * Each line can only contain a single argument.
- * The same style applies to dicts, lists, return calls, etc.
+* Arguments on first line, directly after the opening parenthesis are forbidden when breaking lines.
+
+* The last argument line needs to have a trailing comma (to be nice to the next developer coming in to add something as an argument and minimize VCS diffs in these cases).
+
+* The closing parenthesis or bracket needs to have the same indentation level as the first line.
+
+* Each line can only contain a single argument.
+
+* The same style applies to dicts, lists, return calls, etc.
```

Comment:
There's no need to add an empty line between them, if not instead of creating a single list with X items you are creating X lists with only one item on each